AWSRecreateSG Improvements (#29311)
demisto/content: Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_4.md

#### Scripts

##### AWSRecreateSG

> - Added the **assume_role** optional input in case an roleArn needs to be passed in.

Reviewer: ```suggestion
- Added the **assume_role** optional input in case a roleArn needs to be passed in.
```

> - Updated the script to work even when there are multiple AWS-EC2 integration instances configured.

Reviewer 2: Think this might be a bit better
```suggestion
- Updated the script to work with multiple AWS-EC2 integration instances configured.
```

Author: Shirley already reviewed so I prefer to keep as-is
